physionet-django/project/modelcomponents/access.py

```diff
@@ -167,6 +168,35 @@ class Meta:
         default_permissions = ()
 
 
+class DataSource(models.Model):
+    """
+    Controls all access to project data.
+    """
+    class DataLocation(models.TextChoices):
+        DIRECT = 'DI', _('Direct')
```

Comment:
Do we need this to be translatable? Almost all the options are proper nouns that are not translatable (e.g. AWS S3, AWS Open Data). Also, nothing else in the console is translatable.

Reply:
I agree, we  don't need this to be translatable. To be honest, i was just following what i had seen in Django docs, and had not looked at the import. This is a good lesson for me to understand the code more.